consign v0.1.2 Initialized in /opt/node/mmorpg_got
+ ./app/routes/cadastro.js
+ ./app/routes/index.js
+ ./app/routes/jogo.js
+ ./config/dbConnection.js
+ ./app/models/UsuariosDAO.js
+ ./app/models/jogoDAO.js
+ ./app/controllers/cadastro.js
+ ./app/controllers/index.js
+ ./app/controllers/jogo.js
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8001
    at Server.setupListenHandle [as _listen2] (net.js:1280:14)
    at listenInCluster (net.js:1328:12)
    at Server.listen (net.js:1415:7)
    at Function.listen (/opt/node/mmorpg_got/node_modules/express/lib/application.js:617:24)
    at Object.<anonymous> (/opt/node/mmorpg_got/app.js:5:5)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
Emitted 'error' event at:
    at emitErrorNT (net.js:1307:8)
    at process._tickCallback (internal/process/next_tick.js:63:19)
    at Function.Module.runMain (internal/modules/cjs/loader.js:834:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
consign v0.1.2 Initialized in /opt/node/apps/servidor
+ ./app/routes/index.js
Servidor online
consign v0.1.2 Initialized in /opt/node/mmorpg_got
+ ./app/routes/cadastro.js
+ ./app/routes/index.js
+ ./app/routes/jogo.js
+ ./config/dbConnection.js
+ ./app/models/UsuariosDAO.js
+ ./app/models/jogoDAO.js
+ ./app/controllers/cadastro.js
+ ./app/controllers/index.js
+ ./app/controllers/jogo.js
Servidor online
MongoError: server instance in invalid state undefined
    at Function.MongoError.create (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/error.js:29:11)
    at Server.connect (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/topologies/server.js:341:22)
    at Server.connect (/opt/node/mmorpg_got/node_modules/mongodb/lib/server.js:357:17)
    at open (/opt/node/mmorpg_got/node_modules/mongodb/lib/db.js:222:19)
    at Db.open (/opt/node/mmorpg_got/node_modules/mongodb/lib/db.js:245:44)
    at JogoDAO.atualizadoHabilidades (/opt/node/mmorpg_got/app/models/jogoDAO.js:100:19)
    at Object.module.exports.pergaminhos (/opt/node/mmorpg_got/app/controllers/jogo.js:50:11)
    at /opt/node/mmorpg_got/app/routes/jogo.js:15:36
    at Layer.handle [as handle_request] (/opt/node/mmorpg_got/node_modules/express/lib/router/layer.js:95:5)
    at next (/opt/node/mmorpg_got/node_modules/express/lib/router/route.js:131:13)
    at Route.dispatch (/opt/node/mmorpg_got/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/opt/node/mmorpg_got/node_modules/express/lib/router/layer.js:95:5)
    at /opt/node/mmorpg_got/node_modules/express/lib/router/index.js:277:22
    at Function.process_params (/opt/node/mmorpg_got/node_modules/express/lib/router/index.js:330:12)
    at next (/opt/node/mmorpg_got/node_modules/express/lib/router/index.js:271:10)
    at Immediate._onImmediate (/opt/node/mmorpg_got/node_modules/express-session/index.js:502:7)
/opt/node/mmorpg_got/node_modules/mongodb/lib/utils.js:98
    process.nextTick(function() { throw err; });
                                  ^

TypeError: req.next is not a function
    at done (/opt/node/mmorpg_got/node_modules/express/lib/response.js:955:25)
    at tryRender (/opt/node/mmorpg_got/node_modules/express/lib/application.js:641:5)
    at Function.render (/opt/node/mmorpg_got/node_modules/express/lib/application.js:591:3)
    at ServerResponse.render (/opt/node/mmorpg_got/node_modules/express/lib/response.js:960:7)
    at /opt/node/mmorpg_got/app/models/jogoDAO.js:89:9
    at handleCallback (/opt/node/mmorpg_got/node_modules/mongodb/lib/utils.js:96:12)
    at /opt/node/mmorpg_got/node_modules/mongodb/lib/cursor.js:853:16
    at handleCallback (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/cursor.js:172:5)
    at setCursorNotified (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/cursor.js:517:3)
    at /opt/node/mmorpg_got/node_modules/mongodb-core/lib/cursor.js:590:16
    at queryCallback (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/cursor.js:233:18)
    at /opt/node/mmorpg_got/node_modules/mongodb-core/lib/connection/pool.js:436:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
consign v0.1.2 Initialized in /opt/node/apps/servidor
+ ./app/routes/index.js
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8000
    at Server.setupListenHandle [as _listen2] (net.js:1280:14)
    at listenInCluster (net.js:1328:12)
    at Server.listen (net.js:1415:7)
    at Function.listen (/opt/node/apps/servidor/node_modules/express/lib/application.js:617:24)
    at Object.<anonymous> (/opt/node/apps/servidor/app.js:5:5)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
Emitted 'error' event at:
    at emitErrorNT (net.js:1307:8)
    at process._tickCallback (internal/process/next_tick.js:63:19)
    at Function.Module.runMain (internal/modules/cjs/loader.js:834:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
consign v0.1.2 Initialized in /opt/node/mmorpg_got
+ ./app/routes/cadastro.js
+ ./app/routes/index.js
+ ./app/routes/jogo.js
+ ./config/dbConnection.js
+ ./app/models/UsuariosDAO.js
+ ./app/models/jogoDAO.js
+ ./app/controllers/cadastro.js
+ ./app/controllers/index.js
+ ./app/controllers/jogo.js
Servidor online
MongoError: server instance in invalid state undefined
    at Function.MongoError.create (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/error.js:29:11)
    at Server.connect (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/topologies/server.js:341:22)
    at Server.connect (/opt/node/mmorpg_got/node_modules/mongodb/lib/server.js:357:17)
    at open (/opt/node/mmorpg_got/node_modules/mongodb/lib/db.js:222:19)
    at Db.open (/opt/node/mmorpg_got/node_modules/mongodb/lib/db.js:245:44)
    at JogoDAO.atualizadoHabilidades (/opt/node/mmorpg_got/app/models/jogoDAO.js:100:19)
    at Object.module.exports.pergaminhos (/opt/node/mmorpg_got/app/controllers/jogo.js:50:11)
    at /opt/node/mmorpg_got/app/routes/jogo.js:15:36
    at Layer.handle [as handle_request] (/opt/node/mmorpg_got/node_modules/express/lib/router/layer.js:95:5)
    at next (/opt/node/mmorpg_got/node_modules/express/lib/router/route.js:131:13)
    at Route.dispatch (/opt/node/mmorpg_got/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/opt/node/mmorpg_got/node_modules/express/lib/router/layer.js:95:5)
    at /opt/node/mmorpg_got/node_modules/express/lib/router/index.js:277:22
    at Function.process_params (/opt/node/mmorpg_got/node_modules/express/lib/router/index.js:330:12)
    at next (/opt/node/mmorpg_got/node_modules/express/lib/router/index.js:271:10)
    at Immediate._onImmediate (/opt/node/mmorpg_got/node_modules/express-session/index.js:502:7)
/opt/node/mmorpg_got/node_modules/mongodb/lib/utils.js:98
    process.nextTick(function() { throw err; });
                                  ^

TypeError: req.next is not a function
    at done (/opt/node/mmorpg_got/node_modules/express/lib/response.js:955:25)
    at tryRender (/opt/node/mmorpg_got/node_modules/express/lib/application.js:641:5)
    at Function.render (/opt/node/mmorpg_got/node_modules/express/lib/application.js:591:3)
    at ServerResponse.render (/opt/node/mmorpg_got/node_modules/express/lib/response.js:960:7)
    at /opt/node/mmorpg_got/app/models/jogoDAO.js:89:9
    at handleCallback (/opt/node/mmorpg_got/node_modules/mongodb/lib/utils.js:96:12)
    at /opt/node/mmorpg_got/node_modules/mongodb/lib/cursor.js:853:16
    at handleCallback (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/cursor.js:172:5)
    at setCursorNotified (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/cursor.js:517:3)
    at /opt/node/mmorpg_got/node_modules/mongodb-core/lib/cursor.js:590:16
    at queryCallback (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/cursor.js:233:18)
    at /opt/node/mmorpg_got/node_modules/mongodb-core/lib/connection/pool.js:436:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
consign v0.1.2 Initialized in /opt/node/apps/servidor
+ ./app/routes/index.js
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8000
    at Server.setupListenHandle [as _listen2] (net.js:1280:14)
    at listenInCluster (net.js:1328:12)
    at Server.listen (net.js:1415:7)
    at Function.listen (/opt/node/apps/servidor/node_modules/express/lib/application.js:617:24)
    at Object.<anonymous> (/opt/node/apps/servidor/app.js:5:5)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
Emitted 'error' event at:
    at emitErrorNT (net.js:1307:8)
    at process._tickCallback (internal/process/next_tick.js:63:19)
    at Function.Module.runMain (internal/modules/cjs/loader.js:834:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
consign v0.1.2 Initialized in /opt/node/mmorpg_got
+ ./app/routes/cadastro.js
+ ./app/routes/index.js
+ ./app/routes/jogo.js
+ ./config/dbConnection.js
+ ./app/models/UsuariosDAO.js
+ ./app/models/jogoDAO.js
+ ./app/controllers/cadastro.js
+ ./app/controllers/index.js
+ ./app/controllers/jogo.js
Servidor online
MongoError: server instance in invalid state undefined
    at Function.MongoError.create (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/error.js:29:11)
    at Server.connect (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/topologies/server.js:341:22)
    at Server.connect (/opt/node/mmorpg_got/node_modules/mongodb/lib/server.js:357:17)
    at open (/opt/node/mmorpg_got/node_modules/mongodb/lib/db.js:222:19)
    at Db.open (/opt/node/mmorpg_got/node_modules/mongodb/lib/db.js:245:44)
    at JogoDAO.atualizadoHabilidades (/opt/node/mmorpg_got/app/models/jogoDAO.js:100:19)
    at Object.module.exports.pergaminhos (/opt/node/mmorpg_got/app/controllers/jogo.js:50:11)
    at /opt/node/mmorpg_got/app/routes/jogo.js:15:36
    at Layer.handle [as handle_request] (/opt/node/mmorpg_got/node_modules/express/lib/router/layer.js:95:5)
    at next (/opt/node/mmorpg_got/node_modules/express/lib/router/route.js:131:13)
    at Route.dispatch (/opt/node/mmorpg_got/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/opt/node/mmorpg_got/node_modules/express/lib/router/layer.js:95:5)
    at /opt/node/mmorpg_got/node_modules/express/lib/router/index.js:277:22
    at Function.process_params (/opt/node/mmorpg_got/node_modules/express/lib/router/index.js:330:12)
    at next (/opt/node/mmorpg_got/node_modules/express/lib/router/index.js:271:10)
    at Immediate._onImmediate (/opt/node/mmorpg_got/node_modules/express-session/index.js:502:7)
/opt/node/mmorpg_got/node_modules/mongodb/lib/utils.js:98
    process.nextTick(function() { throw err; });
                                  ^

TypeError: req.next is not a function
    at done (/opt/node/mmorpg_got/node_modules/express/lib/response.js:955:25)
    at tryRender (/opt/node/mmorpg_got/node_modules/express/lib/application.js:641:5)
    at Function.render (/opt/node/mmorpg_got/node_modules/express/lib/application.js:591:3)
    at ServerResponse.render (/opt/node/mmorpg_got/node_modules/express/lib/response.js:960:7)
    at /opt/node/mmorpg_got/app/models/jogoDAO.js:89:9
    at handleCallback (/opt/node/mmorpg_got/node_modules/mongodb/lib/utils.js:96:12)
    at /opt/node/mmorpg_got/node_modules/mongodb/lib/cursor.js:853:16
    at handleCallback (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/cursor.js:172:5)
    at setCursorNotified (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/cursor.js:517:3)
    at /opt/node/mmorpg_got/node_modules/mongodb-core/lib/cursor.js:590:16
    at queryCallback (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/cursor.js:233:18)
    at /opt/node/mmorpg_got/node_modules/mongodb-core/lib/connection/pool.js:436:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
consign v0.1.2 Initialized in /opt/node/apps/servidor
+ ./app/routes/index.js
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8000
    at Server.setupListenHandle [as _listen2] (net.js:1280:14)
    at listenInCluster (net.js:1328:12)
    at Server.listen (net.js:1415:7)
    at Function.listen (/opt/node/apps/servidor/node_modules/express/lib/application.js:617:24)
    at Object.<anonymous> (/opt/node/apps/servidor/app.js:5:5)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
Emitted 'error' event at:
    at emitErrorNT (net.js:1307:8)
    at process._tickCallback (internal/process/next_tick.js:63:19)
    at Function.Module.runMain (internal/modules/cjs/loader.js:834:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
consign v0.1.2 Initialized in /opt/node/mmorpg_got
+ ./app/routes/cadastro.js
+ ./app/routes/index.js
+ ./app/routes/jogo.js
+ ./config/dbConnection.js
+ ./app/models/UsuariosDAO.js
+ ./app/models/jogoDAO.js
+ ./app/controllers/cadastro.js
+ ./app/controllers/index.js
+ ./app/controllers/jogo.js
Servidor online
/bin/up.sh, linha 4: 17656 Terminado               node /opt/node/mmorpg_got/app.js
consign v0.1.2 Initialized in /opt/node/apps/servidor
+ ./app/routes/index.js
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8000
    at Server.setupListenHandle [as _listen2] (net.js:1280:14)
    at listenInCluster (net.js:1328:12)
    at Server.listen (net.js:1415:7)
    at Function.listen (/opt/node/apps/servidor/node_modules/express/lib/application.js:617:24)
    at Object.<anonymous> (/opt/node/apps/servidor/app.js:5:5)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
Emitted 'error' event at:
    at emitErrorNT (net.js:1307:8)
    at process._tickCallback (internal/process/next_tick.js:63:19)
    at Function.Module.runMain (internal/modules/cjs/loader.js:834:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
consign v0.1.2 Initialized in /opt/node/mmorpg_got
+ ./app/routes/cadastro.js
+ ./app/routes/index.js
+ ./app/routes/jogo.js
+ ./config/dbConnection.js
+ ./app/models/UsuariosDAO.js
+ ./app/models/jogoDAO.js
+ ./app/controllers/cadastro.js
+ ./app/controllers/index.js
+ ./app/controllers/jogo.js
Servidor online
/bin/up.sh, linha 4: 18600 Terminado               node /opt/node/mmorpg_got/app.js
consign v0.1.2 Initialized in /opt/node/apps/servidor
+ ./app/routes/index.js
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8000
    at Server.setupListenHandle [as _listen2] (net.js:1280:14)
    at listenInCluster (net.js:1328:12)
    at Server.listen (net.js:1415:7)
    at Function.listen (/opt/node/apps/servidor/node_modules/express/lib/application.js:617:24)
    at Object.<anonymous> (/opt/node/apps/servidor/app.js:5:5)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
Emitted 'error' event at:
    at emitErrorNT (net.js:1307:8)
    at process._tickCallback (internal/process/next_tick.js:63:19)
    at Function.Module.runMain (internal/modules/cjs/loader.js:834:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
consign v0.1.2 Initialized in /opt/node/mmorpg_got
+ ./app/routes/cadastro.js
+ ./app/routes/index.js
+ ./app/routes/jogo.js
+ ./config/dbConnection.js
+ ./app/models/UsuariosDAO.js
+ ./app/models/jogoDAO.js
+ ./app/controllers/cadastro.js
+ ./app/controllers/index.js
+ ./app/controllers/jogo.js
Servidor online
MongoError: server instance in invalid state undefined
    at Function.MongoError.create (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/error.js:29:11)
    at Server.connect (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/topologies/server.js:341:22)
    at Server.connect (/opt/node/mmorpg_got/node_modules/mongodb/lib/server.js:357:17)
    at open (/opt/node/mmorpg_got/node_modules/mongodb/lib/db.js:222:19)
    at Db.open (/opt/node/mmorpg_got/node_modules/mongodb/lib/db.js:245:44)
    at JogoDAO.atulizarPara (/opt/node/mmorpg_got/app/models/jogoDAO.js:120:19)
    at Object.module.exports.pergaminhos (/opt/node/mmorpg_got/app/controllers/jogo.js:50:11)
    at /opt/node/mmorpg_got/app/routes/jogo.js:15:36
    at Layer.handle [as handle_request] (/opt/node/mmorpg_got/node_modules/express/lib/router/layer.js:95:5)
    at next (/opt/node/mmorpg_got/node_modules/express/lib/router/route.js:131:13)
    at Route.dispatch (/opt/node/mmorpg_got/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/opt/node/mmorpg_got/node_modules/express/lib/router/layer.js:95:5)
    at /opt/node/mmorpg_got/node_modules/express/lib/router/index.js:277:22
    at Function.process_params (/opt/node/mmorpg_got/node_modules/express/lib/router/index.js:330:12)
    at next (/opt/node/mmorpg_got/node_modules/express/lib/router/index.js:271:10)
    at Immediate._onImmediate (/opt/node/mmorpg_got/node_modules/express-session/index.js:502:7)
/opt/node/mmorpg_got/node_modules/mongodb/lib/utils.js:98
    process.nextTick(function() { throw err; });
                                  ^

TypeError: req.next is not a function
    at done (/opt/node/mmorpg_got/node_modules/express/lib/response.js:955:25)
    at tryRender (/opt/node/mmorpg_got/node_modules/express/lib/application.js:641:5)
    at Function.render (/opt/node/mmorpg_got/node_modules/express/lib/application.js:591:3)
    at ServerResponse.render (/opt/node/mmorpg_got/node_modules/express/lib/response.js:960:7)
    at /opt/node/mmorpg_got/app/models/jogoDAO.js:89:9
    at handleCallback (/opt/node/mmorpg_got/node_modules/mongodb/lib/utils.js:96:12)
    at /opt/node/mmorpg_got/node_modules/mongodb/lib/cursor.js:853:16
    at handleCallback (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/cursor.js:172:5)
    at setCursorNotified (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/cursor.js:517:3)
    at /opt/node/mmorpg_got/node_modules/mongodb-core/lib/cursor.js:590:16
    at queryCallback (/opt/node/mmorpg_got/node_modules/mongodb-core/lib/cursor.js:233:18)
    at /opt/node/mmorpg_got/node_modules/mongodb-core/lib/connection/pool.js:436:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
consign v0.1.2 Initialized in /opt/node/apps/servidor
+ ./app/routes/index.js
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8000
    at Server.setupListenHandle [as _listen2] (net.js:1280:14)
    at listenInCluster (net.js:1328:12)
    at Server.listen (net.js:1415:7)
    at Function.listen (/opt/node/apps/servidor/node_modules/express/lib/application.js:617:24)
    at Object.<anonymous> (/opt/node/apps/servidor/app.js:5:5)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
Emitted 'error' event at:
    at emitErrorNT (net.js:1307:8)
    at process._tickCallback (internal/process/next_tick.js:63:19)
    at Function.Module.runMain (internal/modules/cjs/loader.js:834:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
/bin/up.sh, linha 7: 10463 Desconexão             node /opt/node/apps/servidor/app.js
